coordinator/states.py

```diff
@@ -141,7 +143,7 @@ def on_entry(self, data):
 
         subscribed = data["subscribed"]
         ready = data["ready"]
-        if ready == subscribed.intersection(ready):
+        if subscribed == subscribed.intersection(ready):
```

Comment:
issubset seems a bit simpler, if I understand correctly what this line is doing

Reply:
Agreed, I've changed it